code/src/java/pcgen/io/exporttoken/SkillToken.java

```diff
@@ -453,18 +453,25 @@ private String getSkillPropValue(Skill aSkill, int property,
 					break;
 
 				case SKILL_EXCLUSIVE_TOTAL:
-					retValue
-						.append(Integer
-							.toString(((aSkill.getSafe(ObjectKey.EXCLUSIVE) || !aSkill.getSafe(ObjectKey.USE_UNTRAINED)) && (SkillRankControl.getTotalRank(pc, aSkill)
-								.intValue() == 0)) ? 0
-								: (SkillRankControl.getTotalRank(pc, aSkill).intValue() + SkillModifier
-									.modifier(aSkill, pc))));
+					retValue.append(Integer
+							.toString((
+									(
+											aSkill.getSafe(ObjectKey.EXCLUSIVE)
+													|| !aSkill.getSafe(ObjectKey.USE_UNTRAINED)) && (
+											SkillRankControl.getTotalRank(pc, aSkill)
+											                .intValue() == 0)) ? 0
+									: (
+									SkillRankControl.getTotalRank(pc, aSkill).intValue() + SkillModifier
+											.modifier(aSkill, pc))));
```

Comment:
This is challenging = and needs a new structure.  Some of this needs to be broken out so it's readable.  Currently it is not.  Not your fault, but if you're going to change the formatting, fix the problem please :)